packages/ui/src/forum/components/threads/ThreadBrowser.tsx

```diff
@@ -25,88 +17,23 @@ export interface ThreadBrowserProps {
   emptyText: string
 }
 
-export const ThreadBrowser = ({
-  label,
-  threads,
-  pageCount,
-  totalCount,
-  isLoading,
-  currentPage,
-  setCurrentPage,
-  emptyText,
-}: ThreadBrowserProps) => {
-  const onPrevClick = () => setCurrentPage(currentPage - 1)
-  const onNextClick = () => setCurrentPage(currentPage + 1)
-
+export const ThreadBrowser = ({ label, threads, isLoading, emptyText }: ThreadBrowserProps) => {
```

Comment:
![thread browser](https://user-images.githubusercontent.com/93924525/171612242-f27c7b99-d2e5-41b1-ae01-09f9976bb592.png)
The new thread browser should also support pagination, therefore the properties: `pageCount`,` totalCount`, `currentPage`, and `setCurrentPage` are still needed.

Please try to reuse some of the previous `ThreadBrowser` logic to implement the new pagination.